sphinx/ext/imgmath.py

```diff
@@ -103,16 +103,17 @@ def generate_latex_macro(image_format: str,
     }
 
     if config.imgmath_use_preview:
-        template_name = 'preview.tex_t'
+        template_name = 'preview.tex'
     else:
-        template_name = 'template.tex_t'
+        template_name = 'template.tex'
 
     for template_dir in config.templates_path:
-        template = path.join(confdir, template_dir, template_name)
-        if path.exists(template):
-            return LaTeXRenderer().render(template, variables)
+        for template_suffix in ('_t', '.jinja'):
```

Comment:
Similarly on the question of [filename-loading-precedence](https://github.com/sphinx-doc/sphinx/pull/12364/files#r1664020965) - perhaps this tuple should be re-ordered:

```suggestion
        for template_suffix in ('.jinja', '_t'):
```

(because we state that we prefer the `.jinja` suffix)